### DISCUSSION TITLE
error: invalid path 'src/?middleware.ts' 에러

### DISCUSSION THREAD

**Discussion by miniposi** [GymHubCommunity/GymHub-FE#159] [ErrorLog] [answered]
### 🚨 에러

> git bash 에서 command를 입력하려 할 때, 자꾸 해당 에러 발생 💢

![image](https://github.com/GymHubCommunity/GymHub-FE/assets/52947668/e114ad00-abc8-4475-92d5-6bcc05482595)

**Comment by miniposi** [comment, marked as answer]
### 원인
분명 파일에는 `src/middleware.ts` 이렇게 되어있는데, 알고보니 `src/\bmiddleware.ts` 이런 식으로 저장이 되어있었다!
(정확한 원인은 모름)

### 해결방안
파일 이름에 특수문자를 원도우 파일 시스템이 인식하지 못해서 발생했기 때문에, 아래 command로 해결!
```
git config core.protectNTFS false
git checkout -f HEAD
```

**Comment by hanseulhee** [comment]
음 왜 그렇게 저장되어있엇지 ??
